docusaurus/docs/operate/morse_migration/claiming.md

```diff
@@ -141,113 +247,171 @@ bring iron tattoo denial extend federal aim best need all prosper draw learn pos
-  collect-morse-accounts Collect account balances and stakes from [morse-state-export-path] JSON file and output to [morse-account-state-path] as JSON
+These subcommands map 1:1 to the different claim messages (see image above) which are used, depending on whether the Morse account being claimed is staked, and if so, as what actor type.
+
+You can run the following command to see all available subcommands:
 
-# TODO_MAINNET(@bryanchriswhite, #1034): Update this once the CLI is available.
+```shell
+poktrolld tx migrate --help
+```
+
+Which should produce output similar to the following:
+
+```shell
+  claim-account
+  claim-application
+  claim-supplier
+  collect-morse-accounts Collect account balances and stakes from [morse-state-export-path] JSON file and output to [morse-account-state-path] as JSON
 ```
 
 :::important
+
 The **`--from=<shannon_key_name>`** flag is used to specify which key in the `poktrolld` keyring is used to sign any given Morse claim message.
 
-**This determines and corresponds to the Shannon "destination" address; i.e., where the Morse account balance will be minted.**
+**This determines and corresponds to the Shannon "destination" address; i.e., where the Morse account balance will be minted/transferred to.**
+
 :::
 
-### Claim an Unstaked/non-actor Account
+### Claim a Basic Morse Account (Unstaked, Non-Actor)
 
 :::note
-If the Morse account is not staked as an application or supplier, it is considered an "unstaked" or "non-actor" account.
+
+If the Morse account is not staked as an Application or Supplier, it is considered an "unstaked" or "non-actor" account.
+
 :::
 
 Claiming an unstaked account will mint the unstaked balance of the Morse account being claimed to the Shannon account which the signer of the `MsgClaimMorseAccount`.
 
-This unstaked balance amount is retrieved from the corresponding onchain `MorseClaimableAccount` which was imported by the foundation (see: [State Export / Transform / Import](./migration.md#state-export--transform--validate)).
+This unstaked balance amount is retrieved from the corresponding onchain `MorseClaimableAccount` which was imported by the foundation.
+
+For example, running the following command:
 
 ```bash
-$ poktrolld migrate claim-account \
+poktrolld migrate claim-account \
   ./pocket-account-8b257c7f4e884e49bafc540d874f33f91436e1dc.json \
   --from app1
-Enter Decrypt Passphrase: 
+```
+
+Should prompt for a passphrase and produce output similar to the following:
+
+```shell
+Enter Decrypt Passphrase:
 MsgClaimMorseAccount {
   "shannon_dest_address": "pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4",
   "morse_src_address": "8B257C7F4E884E49BAFC540D874F33F91436E1DC",
   "morse_signature": "hLGhLRjP6jgP6wgOIaYFxIxT3z4jb4IBDKovMkX5AqUsOqdF+rEIO5aofOKnmYW9BkqL0v2DfUfE3nj25FNhBA=="
 }
-Confirm MsgClaimMorseAccount: y/[n]: 
+Confirm MsgClaimMorseAccount: y/[n]:
 ```
 
-### Claim an Application Staked Account
+### Claim a Morse Application (Staked, Actor)
```

Comment:
Why change this H3 and not the following one?

> ### Claim a Supplier (Servicer) Staked Account

Reply:
I don't fully understand this.

Update the ToC and here is where things stand now:

![Screenshot 2025-03-18 at 2 00 17 PM](https://github.com/user-attachments/assets/a54220ef-29d6-4856-8661-6ef2432bb325)